docs/home/contribute/style-guide.md

```diff
@@ -179,8 +179,12 @@ A list of Kubernetes-specific terms and words to be used consistently across the
 
 <table>
   <tr><th>Term</th><th>Usage</th></tr>
-  <tr><td>Kubernetes</td><td>Kubernetes should always be capitalized.</td></tr>
   <tr><td>Docker</td><td>Docker should always be capitalized.</td></tr>
+  <tr><td>Kubernetes</td><td>Kubernetes should always be capitalized.</td></tr>
+  <tr><td>kubectl</td><td>kubectl should always not be capitalized.</td></tr>
+  <tr><td>Minikube</td><td>Minikube should always be capitalized.</td></tr>
+  <tr><td>Persistent Volumes</td><td>Persistent Volumes should always be capitalized.</td></tr>
+  <tr><td>Salt</td><td>Salt should always be capitalized.</td></tr>
```

Comment:
is there still much Salt referenced? It seems to be decreasing (a large section was deprecated related to it a few months ago) - this worth a call out?